components/usage-api/usage/v1/billing.proto

```diff
@@ -50,3 +53,12 @@ message FinalizeInvoiceRequest {
 
 message FinalizeInvoiceResponse {
 }
+
+message MarkWorkspaceInstanceSessionAsBilledRequest {
+  string instance_id = 1;
+  google.protobuf.Timestamp instance_creation_time = 2;
```

Comment:
What's this field used for?

Reply:
The creation time is for the `From` field for the BillingSession.

Reply:
Ah, so a Session is identified by (id, from). If a second session exists, then the from on this second session will be the `to` for the first, correct?

Might make sense to comment in the proto with this as this may not be obivous. Or at least say that WorkspaceInstance can be split into multiple WorkspaceSession records, each with the above property

Reply:
> If a second session exists, then the from on this second session will be the `to` for the first, correct?

Correct. But I wonder if there is a better way for the `to`. Do you want to hop on Kumo sometime when you're free?

Reply:
Also I changed `instance_creation_time` to `from` since it will be converted before.